src/tiledimage.js

```diff
@@ -771,10 +771,12 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
 
     /**
      * @param {Number} opacity Opacity the tiled image should be drawn at.
+     * @fires OpenSeadragon.TiledImage.event:opacity-change
      */
     setOpacity: function(opacity) {
         this.opacity = opacity;
         this._needsDraw = true;
+        this.raiseEvent('opacity-change');
```

Comment:
Please add documentation for this event (similar to `clip-change` and `bounds-change`), and the other new event. 

Also, it might be good to include the new opacity/composite operation in the event, similar to the `fully-loaded-change` event.